spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/TaskConfigurationProperties.java

```diff
@@ -77,6 +84,17 @@ public void setComposedTaskRunnerUri(String composedTaskRunnerUri) {
 		}
 	}
 
+	public ZoneId getPlatformTimezone() {
+		try {
+			return taskPlatformConfigurationProperties.getTimezone() == null ? null
+					: ZoneId.of(taskPlatformConfigurationProperties.getTimezone());
+		}
+		catch (ZoneRulesException ex) {
```

Comment:
Why are we logging this exception?   Seems like it would give us a false "null" value.

Reply:
This is to ensure that the server runs without problems when the timezone is entered incorrectly, but the corresponding part will be deleted.